sparkpost/transmissions.py

```diff
@@ -74,9 +74,10 @@ def _translate_keys(self, **kwargs):
 
             model['recipients'] = self._extract_recipients(recipients)
 
-        attachments = kwargs.get('attachments', [])
-        model['content']['attachments'] = self._extract_attachments(
-            attachments)
+        attachments = kwargs.get('attachments')
+        if attachments:
```

Comment:
I believe this should wrap line #83 instead of this line (inline_images instead of attachments)